pkg/service/server.go

```diff
@@ -192,10 +199,11 @@ func (mgr *Manager) newPoller() {
 
 func (mgr *Manager) newWorker(message *sqs.Message) {
```

Comment:
This doesn't look like it needs a pointer receiver

Reply:
This is out of bound for this PR as it wasn't added here.
I'll take a look but pretty sure I need to work on a pointer receiver here due to downstream mutations of `ManagerContext`.